staging/src/k8s.io/apiserver/pkg/storage/etcd3/store_test.go

```diff
@@ -226,7 +226,6 @@ func TestGet(t *testing.T) {
 		expectRVTooLarge  bool
 		expectedOut       *example.Pod
 		rv                string
-		expectedRV        string
```

Comment:
field `expectedRV` might be related to the TODO(jpbetz) on line 220? I will defer to @jpbetz.

Reply:
It's fine to remove `expectedRV`. I'm pretty sure there was a WIP version of this where I wanted to test exact RVs and then later I decided to take a more black box approach.